### PR TITLE
support for laravel 7

### PR DESCRIPTION
**Pull request by yaelahan**
proof: Screenshot_20200304_205531

### CODE REVIEW

**Comment by yaelahan** [comment]
for testing add this line into **composer.json** on your laravel project
```
...

"repositories": [
    {
        "type": "vcs",
        "url": "https://github.com/yaelahan/tailwindcss"
    }
],
...
```

and then run 

> composer require laravel-frontend-presets/tailwindcss:dev-master --dev
> php artisan ui tailwindcss --auth
> npm install && npm run dev
> php artisan serve

**Comment by perezale** [comment]
Worked for me!
Tried with php artisan ui tailwindcss (no auth)

**Comment by sjardim** [comment]
Confirm it's working with Auth as well. Thank you!

File: src/TailwindCssPresetServiceProvider.php [hunk 1]
```diff
@@ -2,30 +2,25 @@
...
+        UiCommand::macro('tailwindcss', function ($command) {
             TailwindCssPreset::install();
 
             $command->info('Tailwind CSS scaffolding installed successfully.');
-            $command->info('Please run "npm install && npm run dev" to compile your fresh scaffolding.');
+            $command->comment('Please run "npm install && npm run dev" to compile your fresh scaffolding.');
         });
 
-        PresetCommand::macro('tailwindcss-auth', function ($command) {
+        UiCommand::macro('tailwindcss-auth', function ($command) {
```
Comment (mathieutu) [inline src/TailwindCssPresetServiceProvider.php]:
```suggestion
        AuthCommand::macro('tailwindcss', function ($command) {
```

File: src/TailwindCssPresetServiceProvider.php [hunk 2]
```diff
@@ -2,30 +2,25 @@
 
 namespace LaravelFrontendPresets\TailwindCssPreset;
 
-use Illuminate\Pagination\Paginator;
 use Illuminate\Support\ServiceProvider;
-use Illuminate\Foundation\Console\PresetCommand;
+use Laravel\Ui\UiCommand;
```
Comment (mathieutu) [inline src/TailwindCssPresetServiceProvider.php:6]:
```suggestion
use Laravel\Ui\UiCommand;
use Laravel\Ui\AuthCommand;
```

**Review by mathieutu — CHANGES_REQUESTED** [review]
Hi, 
The auth scaffolding is now done by the ui:auth command (or ui --auth).

As you can see in the follwing lines, we can add a type by adding a macro to the AuthCommand:

https://github.com/laravel/ui/blob/948bd6ad08467b8e9cd03cd2af8d86a3a330bbc0/src/AuthCommand.php#L52-L54

I've made the modifications in suggestions.

File: src/TailwindCssPresetServiceProvider.php (hunk 1 above)
Comment (yaelahan) [reply src/TailwindCssPresetServiceProvider.php]: nice one. thank you for reviewing

File: src/TailwindCssPreset.php [hunk 3]
```diff
@@ -73,11 +75,21 @@ protected static function updateWelcomePage()
         copy(__DIR__.'/tailwindcss-stubs/resources/views/welcome.blade.php', resource_path('views/welcome.blade.php'));
     }
 
-    protected static function updatePagination()
```
Comment (mathieutu) [inline src/TailwindCssPreset.php:76]: So we remove the tailwind pagination? Why? 🤔

**Comment by michaeldyrynda** [comment]
I restored the pagination functionality and tagged as 4.0.0. Thanks!

**Pull request merged**